manage.py

```diff
@@ -59,6 +59,7 @@ def init_hq_python_path():
         'ptop_preindex',
         'sync_prepare_couchdb_multi',
         'sync_couch_views',
+        'celery',
```

Comment:
how about this? https://github.com/dimagi/commcare-hq/pull/10824/commits/1e94a3201de1112cac6698de0545803377a341ad beat now works. sorry for some reason thought beat wasnt working for other reasons